aviatrix/resource_aviatrix_firenet.go

```diff
@@ -572,6 +633,19 @@ func resourceAviatrixFireNetDelete(d *schema.ResourceData, meta interface{}) err
 	fireNet := &goaviatrix.FireNet{
 		VpcID: d.Get("vpc_id").(string),
 	}
+
+	var egressStaticCidrs []string
+	for _, v := range d.Get("egress_static_cidrs").(*schema.Set).List() {
+		egressStaticCidrs = append(egressStaticCidrs, v.(string))
+	}
```

Comment:
You can just check `len(d.Get("egress_static_cidrs").(*schema.Set).List()) != 0`